### PR TITLE
receivers: amended before_record_index receiver

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: invenio_workflows/receivers.py [hunk 1]
```diff
@@ -107,11 +107,12 @@ def index_holdingpen_record(sender, **kwargs):
...
     record["_collections"] = get_record_collections(record)
 
-    # Trigger any before_record_index receivers
-    before_record_index.send(sender.id, json=record)
-
     # Depends on "_collections" being filled correctly for record
     record_index = get_record_index(record)
+
+    # Trigger any before_record_index receivers
+    before_record_index.send(sender.id, json=record, index=index)
```
Comment (jalavik) [inline invenio_workflows/receivers.py]: Should be `record_index`.

Comment (kaplun) [inline invenio_workflows/receivers.py]: f**k

Comment (kaplun) [inline invenio_workflows/receivers.py]: One line: one error.

**Comment by kaplun** [comment]
@jalavik is Travis working on this repo?

**Pull request merged**